nircam_simulator/scripts/catalog_seed_image.py

```diff
@@ -386,11 +386,30 @@ def calcFrameTime(self):
         yd, xd = self.nominal_dims
         # self.frametime = (xd/self.params['Readout']['namp'] + 12.) * (yd + 1) * 10.00 * 1.e-6
         # UPDATED VERSION, 16 Sept 2017
-        colpad = 12
-        rowpad = 2
-        if ((xd <= 8) & (yd <= 8)):
-            rowpad = 3
-        self.frametime = ((1.0 * xd / self.params['Readout']['namp'] + colpad) * (yd + rowpad)) * 1.e-5
+        if 'nircam' in self.params['Inst']['instrument']:
```

Comment:
We should safeguard against uppercase/lowercase issues. This can be achieved with
`if 'nircam' in self.params['Inst']['instrument'].lower()`